src/redis.c

```diff
@@ -441,6 +441,8 @@ static int redis_read(void) /* {{{ */
                       DS_TYPE_GAUGE);
     redis_handle_info(rn->name, rr->str, "memory_lua", NULL, "used_memory_lua",
                       DS_TYPE_GAUGE);
+    redis_handle_info(rn->name, rr->str, "memory_fragmentation_ratio", NULL,
```

Comment:
We avoid such specific *types* and prefer to introduce more generic types that can be reused elsewhere. I'd suggest to add a new `ratio` type and set the *type instance* to `memory_fragmentation`. If the ratio is always in the [0-1] range, the existing `percent` type would also be an option.

Reply:
Hi @octo,

Sure, it makes sense. I'll apply those change to the PR and let you now as soon it's done.

Reply:
Thank you!